src/main/groovy/com/base2/ciinabox/JobHelper.groovy

```diff
@@ -154,6 +154,9 @@ class JobHelper {
         }
         branch(buildBranch)
         wipeOutWorkspace()
+        if (vars.containsKey('subdirectory')) {
```

Comment:
not sure subdirectory is the right name... maybe something like repo_target_dir WDYT?